omnigibson/robots/manipulation_robot.py

```diff
@@ -1289,9 +1301,17 @@ def _handle_assisted_grasping(self):
             if controller.control is None:
                 applying_grasp = False
             elif self._grasping_direction == "lower":
-                applying_grasp = th.any(controller.control < threshold)
+                applying_grasp = (
+                    th.any(controller.control < threshold)
+                    if controller.control_type == ControlType.POSITION
+                    else th.any(controller.control < 0)
+                )
             else:
-                applying_grasp = th.any(controller.control > threshold)
+                applying_grasp = (
+                    th.any(controller.control > threshold)
+                    if controller.control_type == ControlType.POSITION
+                    else th.any(controller.control > 0)
```

Comment:
should this be `>=`?

Reply:
The original behavior is `>` so I just preserve it.